src/tests/unit/tests/injected/visualization/highlight-box-formatter.test.ts

```diff
@@ -17,7 +17,7 @@ describe('HighlightBoxFormatterTests', () => {
 
         expect(config.showVisualization).toBe(true);
         expect(config.failureBoxConfig).toBeNull();
-        expect(config.borderColor).toEqual('#CC0000');
+        expect(config.borderColor).toEqual('#e81123');
```

Comment:
It looks like this capitalization difference is causing the unit test failure, just needs a capital E instead :)

Reply:
Thanks @jlperkins!
You were right, I already made the changes :)